# include <sys/select.h> to avoid:
# error: unknown type name ‘fd_set’;
# in sslh-select.c

--- common.h.orig	2021-08-28 14:33:20.000000000 +0000
+++ common.h	2022-11-11 12:54:51.521742165 +0000
@@ -18,6 +18,7 @@
 #include <stdio.h>
 #include <signal.h>
 #include <sys/socket.h>
+#include <sys/select.h>
 #include <sys/wait.h>
 #include <netinet/in.h>
 #include <arpa/inet.h>
